__tests__/app.test.js

```diff
@@ -149,6 +149,41 @@ describe("GET /api/articles/:article_id/comments", () => {
 });
 })
 
+describe("GET /api/users", () => {
+  test("200: responds with an array of user objects, each with properties of username, name, and avatar_url ", () => {
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then(({ body: { users } }) => {
+      expect(users).toHaveLength(4)
+      users.forEach((user) => {
+        expect(user).toMatchObject({
+          username: expect.any(String), 
+          name: expect.any(String),
+          avatar_url: expect.any(String)
+        })
+      })
+    })
+  })
+  // test("200: responds with an empty array when there are no users", () => {
+  //   return request(app)
+  //   .get("/api/users")
+  //   .expect(200)
+  //   .then(({ body: { users } }) => {
+  //     expect(users).toHaveLength(0)
+  //   })
+  // })
+  test("404: sends an appropriate status and error message when attempting to access a non-existent endpoint", () => {
+    return request(app)
+      .get("/api/puppies")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe('does not exist')
+      })
+  })
+})
```

Comment:
If you already have tested at some other point in your project an incorrect path - then you don't need to check it again here. 

You can test an incorrect path once and in it's own describe block as it doesn't belong on this path!